scripts/cargo-helpers.toml

```diff
@@ -0,0 +1,16 @@
+[env]
+BUILD_ENV = "debug"
+
+[tasks.exec-debug]
+command = "cargo"
+private = true
```

Comment:
This is currently an issue:
[cargo-make] INFO - cargo make 0.21.0
[cargo-make] INFO - Using Build File: Makefile.toml
[cargo-make] INFO - Task: tools
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: init
[cargo-make] INFO - Running Task: atomify
[cargo-make] INFO - Execute Command: "cargo" "make" "exec-debug" "--" "build" "-p" "atomify"
[cargo-make] INFO - cargo make 0.21.0
[cargo-make] INFO - Using Build File: Makefile.toml
[cargo-make] INFO - Task: exec-debug
[cargo-make] INFO - Profile: development
[cargo-make] ERROR - Task exec-debug is private
[cargo-make] WARN - Build Failed.
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
 is this maybe invoked by a shell command or something like that?

Reply:
Ok, I miss a facepalm reaction for this one :sweat_smile:

Reply:
targets, that are executed, by shell commands may not be private

Reply:
Fix is pushed.